[Bug]: Maven not found

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hello i got a issue when i try to compile....the error is something like cannot reach https://jcenter.bintray.com/com/facebook/android/facebook-android-sdk/maven-metadata.xml

neither of versions works (14,15+)....i used the first one buy today is not working.....after that i tried to cahnge to 15+ and still not working.....

can you confirm is problem about the central repository or package?

ERROR: (The error is the same with 15.+)



FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve com.facebook.android:facebook-android-sdk:14.+.
     Required by:
         project :app > project :facebook_app_events
      > Failed to list versions for com.facebook.android:facebook-android-sdk.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/android/facebook-android-sdk/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/facebook/android/facebook-android-sdk/maven-metadata.xml'.
               > Read timed out

### Expected Behavior

Compile

### Steps To Reproduce

Just try tu running the app

### Platform

Android

### Flutter Doctor

```shell
[√] Flutter (Channel stable, 3.3.2, on Microsoft Windows [Version    10.0.19044.2130], locale en-US)
    • Flutter version 3.3.2 on channel stable at C:\src\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision e3c29ec00c (7 weeks ago), 2022-09-14
      08:46:55 -0500
    • Engine revision a4ff2c53d8
    • Dart version 2.18.1
    • DevTools version 2.15.0

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at C:\Users\Amilkar\AppData\Local\Android\sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.12+7-b1504.28-7817840)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[X] Visual Studio - develop for Windows
    X Visual Studio not installed; this is necessary for Windows development.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components

[√] Android Studio (version 2021.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+7-b1504.28-7817840)

[√] VS Code (version 1.72.2)
    • VS Code at C:\Users\Amilkar\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (4 available)
    • sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 13 (API 33) (emulator)
    • Windows (desktop)            • windows       • windows-x64    • Microsoft Windows [Version 10.0.19044.2130]
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 107.0.5304.87
    • Edge (web)                   • edge          • web-javascript • Microsoft Edge 107.0.1418.24

[√] HTTP Host Availability
    • All required HTTP hosts are available
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+## 0.18.1
+- Fixing version syntax in gradle.
+
 ## 0.18.0
 - Updating Facebook SDK  dependency to follow all `15.x` versions.
 

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -41,5 +41,15 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.facebook.android:facebook-android-sdk:15.+'
+    // # READ THIS
+    // When updating the Facebook SDK. Please note these guidelines to avoid
+    // conflicts with other plugins that also rely on the FBSDK.
+    //
+    // - Only specify the major version of the SDK. 
+    // - Do **not** specify the `minor` or `patch` version.
+    //
+    // About the version syntax:
+    //  - https://github.com/oddbit/flutter_facebook_app_events/pull/189#discussion_r768267841
+    //  - https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402
+    implementation 'com.facebook.android:facebook-android-sdk:[15.0,16.0)'
 }

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,7 +1,7 @@
 name: facebook_app_events
 description: Flutter plugin for Facebook App Events, an app measurement
   solution that provides insight on app usage and user engagement in Facebook Analytics.
-version: 0.18.0
+version: 0.18.1
 homepage: https://github.com/oddbit/flutter_facebook_app_events
 
 environment:

